### PR TITLE
INBA-351

### PR DESCRIPTION
**Pull request by cporter-amida**
Update Indaba Docker documentation

https://jira.amida-tech.com/browse/INBA-351

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,37 +41,143 @@ Indaba puts your stakeholder and expert network at your fingertips. It converts
 
 1. Set up a Docker Postgres container:
```
Comment (Perry5) [inline README.md]: Is there any installation for Docker that needs to happen before continuing with the steps?

**Comment by Perry5** [comment]
Also, just for consistency sake since we already have the set up process in this wiki: https://paper.dropbox.com/doc/Indaba-Set-up-for-new-developers-Mac-5NzKPSGxbHOUXomHFaEKy

Do you think it'll be better to have the docker set up instructions in the same document and link to it from greyscale. I guess it won't hurt to have it in both places and just have a link to this doc from the readme. The reason being because the Paper doc is more updated on the greyscale set up.

**Comment by jmfadeley** [comment]
@cporter-amida There's a really good chance you want to merge this to develop branch and not master.

**Comment by Perry5** [comment]
Also, yes, please change from master to develop

**Comment by jmfadeley** [comment]
Any forward progress here?

**Comment by jmfadeley** [comment]
Closed due to wrong branch.

**Pull request closed without merging**